SMDS_3/include/CGAL/Mesh_complex_3_in_triangulation_3.h

```diff
@@ -351,6 +351,14 @@ class Mesh_complex_3_in_triangulation_3
     far_vertices_.clear();
   }
 
+  /** sets the internal triangulation to \p tr
+  */
+  void set_triangulation(const Triangulation& tr)
+  {
+    tr_ = tr;
+    rescan_after_load_of_triangulation();
+  }
+
```

Comment:
add a version doing a move:

```suggestion
  void set_triangulation(const Triangulation& tr)
  {
    tr_ = tr;
    rescan_after_load_of_triangulation();
  }
  /** sets the internal triangulation to \p tr
  */
  void set_triangulation(Triangulation&& tr)
  {
    tr_ = std::move(tr);
    rescan_after_load_of_triangulation();
  }

```